File: Raw_Inventarios/temp/extract.js
const fs = require('fs');
const pdf = require('pdf-parse');

const dataBuffer = fs.readFileSync('../Lista de Precios ILVA ENERO 2026 Contado ( B ).pdf');

pdf(dataBuffer).then(function(data) {
    fs.writeFileSync('output.txt', data.text);
    console.log('PDF extracted successfully to output.txt');
}).catch(err => {
    console.error('Error extracting PDF:', err);
});

File: Raw_Inventarios/temp/extract.py
import pdfplumber
import sys

pdf_path = '../Lista de Precios ILVA ENERO 2026 Contado ( B ).pdf'
text_lines = []

try:
    with pdfplumber.open(pdf_path) as pdf:
        for page in pdf.pages:
            text = page.extract_text()
            if text:
                text_lines.append(text)
                
    with open('output.txt', 'w', encoding='utf-8') as f:
        f.write('\n---PAGE---\n'.join(text_lines))
        
    print("Extraction successful")
except Exception as e:
    print(f"Error: {e}")
